### PR TITLE
Deduplicate dartanalyzer output lines, and increase timeout.

### LINKED ISSUE #517: dartanalyzer produces a lot of duplicate rows
Debugging #515 highlighted an interesting thing with `dartanalyzer`: running it with normal output will produce `7092 lints and 2 hints found` (`7094` total).

Running it with `--format machine` produced `1443845` lines of output, but after sorting the duplicates, it was only `7094` lines of output. We cut the analysis because the output process is above 100k lines, but we could rather deduplicate the lines as we read them.

Also, reporting issue to `dartanalyzer` repo.

### PR DESCRIPTION
**Pull request by isoos**
Fixes #517, improves #515.

### CODE REVIEW

File: lib/src/utils.dart [hunk 1]
```diff
@@ -65,6 +66,9 @@ Future<ProcessResult> runProc(
   var items = await Future.wait(<Future<Object>>[
     process.exitCode,
     byteStreamSplit(process.stdout).forEach((outLine) {
+      if (deduplicate && stdoutLines.contains(outLine)) {
```
Comment (jonasfj) [inline lib/src/utils.dart]: Why is this necessary? it seems like something that should be fixed else where...

Comment (isoos) [reply lib/src/utils.dart]: I've filed an issue for dartanalyzer, but I also want us to unblock from waiting for it to happen.

Comment (kevmoo) [reply lib/src/utils.dart]: Perhaps add a todo here w/ a pointer to the issue?
